src/System.Runtime.Analyzers/Core/SpecifyCultureInfo.cs

```diff
@@ -71,9 +71,9 @@ public override void Initialize(AnalysisContext analysisContext)
                             oaContext.ReportDiagnostic(
                                 invocationExpression.Syntax.CreateDiagnostic(
                                     Rule,
-                                    targetMethod.ToDisplayString(SymbolDisplayFormat.CSharpErrorMessageFormat),
-                                    oaContext.ContainingSymbol.ToDisplayString(SymbolDisplayFormat.CSharpErrorMessageFormat),
-                                    correctOverload.ToDisplayString(SymbolDisplayFormat.CSharpErrorMessageFormat)));
+                                    targetMethod.ToDisplayString(SymbolDisplayFormats.ShortSymbolDisplayFormat),
+                                    oaContext.ContainingSymbol.ToDisplayString(SymbolDisplayFormats.ShortSymbolDisplayFormat),
+                                    correctOverload.ToDisplayString(SymbolDisplayFormats.ShortSymbolDisplayFormat)));
                         }
```

Comment:
`SymbolDisplayFormat.CSharpErrorMessageFormat` is defined by Roslyn and we did not want to use the display format with the name `CSharp` in it language agnostic part. Also `CSharpErrorMessageFormat` display format prints the namespace when displaying a symbol which again made the diagnostic message unnecessarily long. Hence the use of `SymbolDisplayFormats.ShortSymbolDisplayFormat`, which fits our exact need